conf/airframes/examples/ardrone2_FPguided_gazebo.xml

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
+<airframe name="ardrone2">
+
+  <firmware name="rotorcraft">
+    <target name="ap" board="ardrone2"/>
+
+    <target name="nps" board="pc">
+      <module name="fdm" type="gazebo"/>
+    </target>
+
+    <define name="USE_SONAR" value="TRUE"/>
+
+	<!-- Subsystem section -->
+    <module name="telemetry" type="transparent_udp"/>
+    <module name="radio_control" type="datalink"/>
+    <module name="motor_mixing"/>
+    <module name="actuators" type="ardrone2"/>
+    <module name="imu" type="ardrone2"/>
+    <!-- gps: "ublox" or change to "sirf" for usage with parrot flight recorder -->
+    <module name="gps" type="ublox"/>
+    <module name="stabilization" type="int_quat"/>
+    <module name="ahrs" type="int_cmpl_quat"/>
+    <module name="ins" type="extended"/>
+    <module name="bat_voltage_ardrone2"/>
+    <!-- remove the gps_ubx_ucenter module if you use the sirf gps (flight recorder) -->
+    <module name="gps" type="ubx_ucenter"/>
+    <module name="send_imu_mag_current"/>
+    <module name="air_data"/>
+    <module name="geo_mag"/>
+    <!--module name="logger_file"/-->
+    <module name="video_thread">
+    </module>
+    <module name="cv_colorfilter">
+      <define name="COLORFILTER_CAMERA" value="front_camera"/>
+      <define name="COLORFILTER_SEND_OBSTACLE" value="TRUE"/>
+    </module>
+
+    <module name="video_rtp_stream">
+      <define name="VIEWVIDEO_CAMERA" value="front_camera"/>
+      <define name="VIEWVIDEO_DOWNSIZE_FACTOR" value="4"/>
+      <define name="VIEWVIDEO_QUALITY_FACTOR" value="60"/>
+    </module>
+
+    <module name="flight_plan_guided"/>
```

Comment:
see https://github.com/paparazzi/paparazzi/blob/master/conf/flight_plans/demo_gvf.xml#L35 for example